GameDev/PlayState.h

```diff
@@ -27,7 +27,7 @@ class PlayState :
 
 		void HandleMouseEvents(SDL_Event mainEvent);
 		void HandleKeyEvents(std::unordered_map<SDL_Keycode, bool>* _events);
-		void Update(float dt);
+		void Update(float);
```

Comment:
dit zorgt niet voor compile errors enzo? Waarom dt weg? Dat maakt duidelijk waar het voor bedoeld is, imho